nextflow.config

```diff
@@ -51,11 +55,19 @@ params {
 
     // Vembrane table options
     tsv                         = true
-    allele_fraction             = 'FORMAT_AD'
+    allele_fraction             = null
     annotation_fields           = 'all'
-    format_fields               = 'GT AD[0] AD[1] DP'
+    format_fields               = 'GT AF AD[0] AD[1] DP'
```

Comment:
In the nextflow parameters, the `--allele_fraction` parameter automatically sets the format field for calculating allele fraction, so `AF` does not need to be included in `--format_fields` parameter. If you use VCF files with AF format fiels, set `--allele_fraction = 'FORMAT_AF'`.